src/malloc.c

```diff
@@ -0,0 +1,6302 @@
+#define ENOMEM 0
```

Comment:
Just add some comments here to clarify the boundary between our config lines and the dlmalloc source.